### PR TITLE
Added job to create elastic indexes when a new wiki is created

### PR DESCRIPTION
**Pull request by bereket-WMDE**
[Bug: T285541](https://phabricator.wikimedia.org/T285541)

### CODE REVIEW

File: app/Http/Controllers/WikiController.php [hunk 1]
```diff
@@ -108,6 +116,8 @@ public function create(Request $request): \Illuminate\Http\Response
         $this->dispatch(new ProvisionWikiDbJob(null, null, 10));
         $this->dispatch(new ProvisionQueryserviceNamespaceJob(null, 10));
 
+        $this->dispatch(new ElasticSearchIndexInit($wiki->domain));
```
Comment (bereket-WMDE) [inline app/Http/Controllers/WikiController.php:119]: @toban 
Do we need to create this class somewhere ?

Comment (bereket-WMDE) [reply app/Http/Controllers/WikiController.php:119]: i guess yes. since psalm is failing the lint test

Comment (toban) [reply app/Http/Controllers/WikiController.php:119]: Looks like we might just need to import it!

**Review by toban — APPROVED** [review]

File: app/Http/Controllers/WikiController.php [hunk 2]
```diff
@@ -79,6 +80,14 @@ public function create(Request $request): \Illuminate\Http\Response
                 'value' => Str::random(64),
             ]);
 
+            // Enable elasticsearch for new wikis by default
+            // T285541
+            WikiSetting::create([
```
Comment (addshore) [inline app/Http/Controllers/WikiController.php:85]: For now I don't think we want to do this!
We should leave the default to be false.
We do however still want to be able to call the API that we made, when the thing is disabled.

**Review by addshore — CHANGES_REQUESTED** [review]

**Review by addshore — APPROVED** [review]

**Pull request merged**